File: src/android/com/kuackmedia/androidauto/models/AlbumTracks.kt
package com.kuackmedia.androidauto.models

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class AlbumTracks(
  override val id: String,
  override val itemType: String,
  override val itemStyle: String = "list",
  override val score: Double? = null,
  val upc: String,
  val title: String,
  @Json(name = "subTitle") val subTitle: String?,
  val releaseType: String?,
  val lenght: String,
  val tracksQty: Int?,
  val releaseDate: String?,
  val active: Boolean?,
  val images: List<CoverImage>,
  val artists: List<Artist>?,
  val tracks: TracksContainer,
  val imageColorInfo: Any?
) : MediaItem()

File: src/ios/models /AlbumTracks.swift
import Foundation

struct AlbumTracks: Codable {
    let id: String
    let itemType: String
    let itemStyle: String
    let upc: String
    let title: String
    let subTitle: String?
    let releaseType: String?
    let lenght: String
    let tracksQty: Int
    let releaseDate: String
    let active: Bool
    let images: [CoverImage]
    let artists: [Artist]
    let tracks: TracksContainer
    let imageColorInfo: AnyCodable?
}
